disperser/batcher/metrics.go

```diff
@@ -133,3 +151,8 @@ func (g *Metrics) Start(ctx context.Context) {
 		log.Error("prometheus server failed", "err", err)
 	}()
 }
+
+func (e *EncodingStreamerMetrics) UpdateEncodedBlobs(count int, size uint) {
```

Comment:
Can the size go above 2GB? If so, needs int64

Reply:
good call